docs/docs/10.4-test-utils.md

```diff
@@ -48,6 +48,12 @@ ReactComponent renderIntoDocument(
 ```
 
 Render a component into a detached DOM node in the document. **This function requires a DOM.**
+> Note:
+>
+> You will need to have `window`, `window.document` and `window.document.createElement`
+ globally available **before** you import React. Otherwise React will think it can't access the DOM and methods like `setState` won't work.  
+>
+> See [#4881](https://github.com/facebook/react/issues/4881) and [#4592](https://github.com/facebook/react/issues/4592) for more information.
```

Comment:
And probably just remove these references. It's not super useful since you already explained the reason.